cost/azure/reserved_instances/mca_recommendations/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v2.16
+
+- Renamed `Reserved Instance` policy_set from info block to `Reserved Instances`.
```

Comment:
The feedback I provided for the other policy also applies to the CHANGELOG entries here.

Reply:
Addressed!